admin/api/cmd/server/options.go

```diff
@@ -107,12 +194,17 @@ func (opts *Options) Run(ctx context.Context) error {
 
 	logger.Info("Running server", "port", opts.Port)
 	mux := http.NewServeMux()
-	mux.Handle("GET /admin/helloworld", handlers.HelloWorldHandler())
+	mux.Handle(MuxPattern(http.MethodGet, "helloworld"), handlers.HelloWorldHandler())
```

Comment:
nit: you probably want the path handling to be done in a robust manner, as opposed to a string prefix

```
	// Root mux
	rootMux := http.NewServeMux()

	// Submux for /api/v1
	apiV1Mux := http.NewServeMux()
	apiV1Mux.HandleFunc("/", apiV1Handler)

	// Submux for /api/v2
	apiV2Mux := http.NewServeMux()
	apiV2Mux.HandleFunc("/", apiV2Handler)

	// Mount submuxes under prefixes
	rootMux.Handle("/api/v1/", http.StripPrefix("/api/v1", apiV1Mux))
	rootMux.Handle("/api/v2/", http.StripPrefix("/api/v2", apiV2Mux))
```

Reply:
ty. done